### PR TITLE
Hang in there - SE M2

### PR DESCRIPTION
**Pull request by hfaerber**
# Curriculum PR Template

### Does this PR close any issues?
This PR, together with [this PR updating the boilerplate repo](https://github.com/turingschool-examples/hang-in-there-boilerplate/pull/399), closes issue #149. 

### Description
- Brings FE Hang-In-There project into SE curriculum as first M2 solo
- Revises spec to present as solo, not paired
- Adds 2 iterations to:
  - allow students to build some htmls/css (instead of just using pre-exisiting)
  - allow for data manipulation and data display on the DOM
- Reformats spec to somewhat follow BE convention of different pages for rubric, peer code review and submission.

### Notes
- I did not create separate pages for each iteration.  I think the current dropdown style will benefit students as they work since various comp photos are found within those dropdowns.  I dont want them having to dig through lots of links to find those if they need to reference again in a later iteration. 
- I did not revise the rubric to match the more traditional style of BE rubrics.  This rubric was recently revamped by the FE team and is a good fit for guiding students through this FE project and its competencies.  I don't thing veering from the usual BE rubric style will be a problem in any way for students or instructors.  If need, we can type of some instructor notes to help provide more framework for grading.  Leaving it as-is also saves some time.
- We should be prepared to be flexible on the functionality requirements here as we gage how much/little students struggle through this work.
- The iterations I added have students contributing new code but doing similar things to what has already been done so they have a sort of template they can work from within the pre-existing code.  This gets them the needed practice without being way too hard (hopefully).
- The final iteration (deleting posters) could be optional if we find they are running out of time.
- This project will be the first week and a half of SE2, they'll start learning FE content and apply it in this lesson.

### What questions do you have/what do you want feedback on? (optional)
- Any general feedback is great
- Feedback on if clarity is lacking in any of the spec directions/info
- Feedback around if we think these expectations are feasible and appropriate in their difficulty

### CODE REVIEW

File: module2/projects/hang-in-there/index.md [hunk 1]
```diff
@@ -0,0 +1,407 @@
+---
+title: Hang In There - Pair
```
Comment (epintozzi) [inline module2/projects/hang-in-there/index.md:2]: Should say Solo not Pair?

Comment (hfaerber) [reply module2/projects/hang-in-there/index.md]: Thank you!  Just pushed up a fix for that.

**Review by niksseif — COMMENTED** [review]
Thanks for working on this, Heather. I left a few suggestions for you. I’m not sure why the project images aren’t showing on my end, but maybe we can create an assets folder in the project directory specifically for the project. Our current assets folder is quite large, and it’s difficult to find all the images there.🤷🏻‍♀️

Also, the link to this project is commented out in the index.md file.

File: module2/projects/hang-in-there/index.md [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+**By EOD on Kick Off Day:** 
+
+* Read the entire project spec and rubric.  Post questions in slack.
+* Add your project manager as a collaborator to your forked repository.
+</section>
+
+<section class="dropdown">
+### Iteration 0 - Main Page
+
+![homepage](../../assets/images/projects/hang-in-there/homepage.png)
```
[the remark this answers is not in the document]
Comment (hfaerber) [reply module2/projects/hang-in-there/index.md:96]: fixed!

File: module2/projects/hang-in-there/index.md [hunk 3]
```diff
@@ -0,0 +1,407 @@
...
+
+- When the page loads, users should see a poster with a randomly selected image, title, and quote
+- Every time the user clicks the Show Random Poster button, a new random poster is displayed.
+</section>
+
+<section class="dropdown">
+### Iteration 1 - Switching Views
+
+Form view:
+![form](../../assets/images/projects/hang-in-there/form.png)
```
[the remark this answers is not in the document]
Comment (hfaerber) [reply module2/projects/hang-in-there/index.md:106]: fixed

File: module2/projects/hang-in-there/index.md [hunk 4]
```diff
@@ -0,0 +1,407 @@
...
+
+Saved posters view:
+![saved posters](../../assets/images/projects/hang-in-there/saved.png)
```
[the remark this answers is not in the document]
Comment (hfaerber) [reply module2/projects/hang-in-there/index.md:109]: fixed

File: module2/projects/hang-in-there/index.md [hunk 5]
```diff
@@ -0,0 +1,407 @@
...
+We aren't actually navigating different pages of a website here. Instead, we will fake that navigation by hiding and unhiding entire sections. Go check out the HTML and CSS files to see how the form and saved posters sections are being hidden in the first place._  
+
+_Consider how you might use arguments/parameters to make dynamic reusable functions to control what is shown/hidden._
+</section>
+
+<section class="dropdown">
+### Iteration 2 - Creating a New Poster
+
+Result after clicking Show My Poster button:
+![result](../../assets/images/projects/hang-in-there/form-result.png)
```
[the remark this answers is not in the document]
Comment (hfaerber) [reply module2/projects/hang-in-there/index.md:126]: fixed

**Pull request merged**